main.py

```diff
@@ -1,21 +1,24 @@
-from ultralytics.utils.tlc.detect.model import TLCYOLO  # noqa: E402
+from ultralytics.utils.tlc import TLCYOLO
+
 from ultralytics.utils.tlc.detect.settings import Settings
 
-model = TLCYOLO("yolov8n.pt")  # initialize
+model = TLCYOLO("yolov8s.pt")  # initialize
 
 # Set 3LC specific settings
 settings = Settings(
-    image_embeddings_dim=2,
-    collection_epoch_start=0,
-    collection_epoch_interval=2,
+    image_embeddings_dim=3,
+    # collection_epoch_start=1,
```

Comment:
Are these intentionally commented out?

Reply:
Removed the files instead! Perhaps we should have some example scripts under `ultralytics/utils/tlc/examples`.